src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -130,19 +133,41 @@ export const TablesUserList: React.FC<TablesUserListProps> = ({
     [recentVenueUsers, user, venueName, videoRoom]
   );
 
+  const itemsToObjectByRefReducer = useCallback(
```

Comment:
`itemsToObjectByRefReducer` -> `usersAtTableReducer` ?

Reply:
Looks like this has a return value, maybe it should be using `useMemo` instead.

Reply:
@denisdimitrov this is a function that is used in `arr.reduce()` a bit later - it takes a table and accumulate the result in the accumulator. I believe we can leave `useCallback` here. what do you think?

Reply:
@sunny-viktoryia Is this function 'closing around' internal state? If so, `useCallback` seems fine. If not, then should be extracted out of the component body into a `utils/*` function or similar.

Reply:
@0xdevalias, the function uses `recentVenueUsers`  and `venueName`. So i'd leave it here.

Reply:
I think the correct way would be to use `useMemo` because it will memo the result, not the function. That way if the dep array doesn't change the value will be returned without the calculations. However this is nitpicking and I don't have a strong opinion about it.

Reply:
@denisdimitrov hmm but it does not have the result by the moment of its creation (if you do not mean the function as the result). It has a function which will produce the result once you pass the arguments.
we could use `useMemo` but it would look like something like this:
```js
useMemo(() => (obj, table) => ({ ... result }), [dependencies])
```
and the current version is:
```js
useCallback((obj, table) => ({ ... result }), [dependencies])
```

Reply:
@sunny-viktoryia That's true and makes sense. The function that's using this should be using memo, and it is. Looks good to me. :)

Reply:
@denisdimitrov cool! glad we've sorted that out. 🙃